### PR TITLE
Serialize Copyright info while publishing a package

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

Add `Copyright Holder` and `Copyright Year` to package serialization file
Update `Greg` to `2.1.8012.24290`

pub2

Snapshot of Package file (`pkg.json`)

Screen Shot 2021-12-08 at 2 37 16 PM

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs [hunk 1]
```diff
@@ -793,6 +793,8 @@ private void ClearAllEntries()
             this.Assemblies = new List<PackageAssembly>();
             this.SelectedHosts = new List<String>();
             this.SelectedHostsString = string.Empty;
+            this.copyrightHolder = "";
+            this.copyrightYear = "";
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs:797]: Let's using string.empty

**Review by QilongTang — COMMENTED** [review]
LGTM with one comment

File: src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs (hunk 1 above)
Comment (zeusongit) [reply src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs:797]: We have used "" in all the variables above this too, do you want me to update those as well?

Comment (zeusongit) [reply src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs]: Done

Comment (QilongTang) [reply src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs]: @zeusongit That would be even better, thanks!

**Review by QilongTang — APPROVED** [review]

**Pull request merged**